i.sentinel_1.change/i.sentinel_1.change.html

```diff
@@ -0,0 +1,32 @@
+<h2>DESCRIPTION</h2>
+
+<em>i.sentinel_1.change</em> is a GRASS GIS addon that extracts areas of changed backscatter values from
+two Sentinel-1 input scenes. The input backscatter values are expected in dB scale.
+The addon calculates two polarisation ratios (date2_vv/date1_vv and date2_vh/date1_vh),
+smoothes them with a median filter in  <a href="r.neighbors.html">r.neighbors</a>
```

Comment:
```suggestion
smoothes them with a median filter in  <a href="https://grass.osgeo.org/grass-stable/manuals/r.neighbors.html">r.neighbors</a>
```

In addon manual pages, links to GRASS-core modules shall be full URLs.